plugins/sync/src/test/kotlin/app/aaps/plugins/sync/garmin/GarminPluginTest.kt

```diff
@@ -242,20 +246,20 @@ class GarminPluginTest: TestBase() {
                 .map(Instant::ofEpochMilli)
                 .mapIndexed { idx, ts -> createGlucoseValue(ts, 100.0+(10 * idx)) }.reversed()}
         assertEquals(
-            """[{"_id":"100000","device":"RANDOM","deviceString":"1970-01-01T00:00:10Z","sysTime":"1970-01-01T00:00:10Z","unfiltered":90.0,"date":10000,"sgv":120,"delta":10,"direction":"Flat","noise":4.5,"units_hint":"mmol"}]""",
+            """[{"_id":"100000","device":"RANDOM","deviceString":"1970-01-01T00:00:10Z","sysTime":"1970-01-01T00:00:10Z","unfiltered":90.0,"date":10000,"sgv":120,"delta":10,"direction":"Flat","noise":4.5,"units_hint":"mmol","iob":0.0,"tbr":0,"cob":0.0}]""",
```

Comment:
Can you add some values, e.g.
whenever(loophub.carbsOnboard).thenReturn(12.3)
whenever(loophub.insulinBasalOnboard).thenReturn(2.71)
whenever(loopHub.temporaryBasal).thenReturn(0.8)

Reply:
Great and thanks!